diff --git a/src/base/ver/verStream.c b/src/base/ver/verStream.c
--- a/src/base/ver/verStream.c
+++ b/src/base/ver/verStream.c
@@ -27,9 +27,9 @@ ABC_NAMESPACE_IMPL_START
 ///                        DECLARATIONS                              ///
 ////////////////////////////////////////////////////////////////////////
 
-#define VER_BUFFER_SIZE        1048576    // 1M  - size of the data chunk stored in memory
-#define VER_OFFSET_SIZE          65536    // 64K - load new data when less than this is left
-#define VER_WORD_SIZE            65536    // 64K - the largest token that can be returned
+#define VER_BUFFER_SIZE        104857600    // 1M  - size of the data chunk stored in memory
+#define VER_OFFSET_SIZE          6553600    // 64K - load new data when less than this is left
+#define VER_WORD_SIZE            6553600    // 64K - the largest token that can be returned
 
 #define VER_MINIMUM(a,b)       (((a) < (b))? (a) : (b))
